src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Helpers/AdalIdHelper.cs

```diff
@@ -98,8 +100,7 @@ internal static class AdalIdHelper
         public static string GetAdalVersion()
         {
             string fullVersion =  typeof (AdalIdHelper).GetTypeInfo().Assembly.FullName;
-            Regex regex = new Regex(@"Version=[\d]+.[\d+]+.[\d]+.[\d]+");
-            Match match = regex.Match(fullVersion);
+            Match match = AdalVersionRegex.Match(fullVersion);
```

Comment:
If this is performed a lot, then a faster approach would be to do:
if (AdalVersionRegex.IsMatch(fullVersion)
{
  return fullVersion.Substring("Version".Length);
}
}